Domain.Sql/ReadModelDatabaseInitializer.cs

```diff
@@ -32,6 +38,10 @@ public void InitializeDatabase(TDbContext context, int dbSizeInGB, string editio
             if (context.IsAzureDatabase())
             {
                 context.CreateAzureDatabase(dbSizeInGB, edition, serviceObjective);
+                if (!string.IsNullOrWhiteSpace(login) && !string.IsNullOrWhiteSpace(readonlyUser))
+                {

```

Comment:
If only one of these is set, nothing happens, which indicates to me that they need to be both set, or it's an error condition. At this point I think the number of parameters may be making the method harder to understand.